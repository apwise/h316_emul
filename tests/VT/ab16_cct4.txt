ptr ../../tapes/VT/AB16-CCT4_slst.ptp
g'1
cl
sbi 1000000
g'1000
a
ss1 1
sbi 1000
cont
a
ss2 1
sbi 1000
cont
a
ss3 1
sbi 1000
cont
a
ss4 1
sbi 1000
cont
a
ss1 0
sbi 1000
cont
a
ss2 0
sbi 1000
cont
a
ss3 0
sbi 1000
cont
a
ss4 0
limit 680832202
cont
q

