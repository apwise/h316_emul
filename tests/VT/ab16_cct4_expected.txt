
MON> ptr ../../tapes/VT/AB16-CCT4_slst.ptp
MON> g'1

MON> cl
MON> sbi 1000000
MON> g'1000


AB16-CCT4   APR 24, 73  REV H 
SENS SW TO A REG 8 TIMES
MON> a
A: 0x0000 '000000 0.000.000.000.000.000
MON> ss1 1
MON> sbi 1000
MON> cont

MON> a
A: 0x8000 '100000 1.000.000.000.000.000
MON> ss2 1
MON> sbi 1000
MON> cont

MON> a
A: 0xc000 '140000 1.100.000.000.000.000
MON> ss3 1
MON> sbi 1000
MON> cont

MON> a
A: 0xe000 '160000 1.110.000.000.000.000
MON> ss4 1
MON> sbi 1000
MON> cont

MON> a
A: 0xf000 '170000 1.111.000.000.000.000
MON> ss1 0
MON> sbi 1000
MON> cont

MON> a
A: 0x7000 '070000 0.111.000.000.000.000
MON> ss2 0
MON> sbi 1000
MON> cont

MON> a
A: 0x3000 '030000 0.011.000.000.000.000
MON> ss3 0
MON> sbi 1000
MON> cont

MON> a
A: 0x1000 '010000 0.001.000.000.000.000
MON> ss4 0
MON> limit 680832202
MON> cont

DONE
CP IS  316   MMSZ 32K 

PS  0001
PS  0002
0700000003: limit reached

MON> q
